### PR TITLE
Install mapzen fork of apiaxle-api

### PR DESCRIPTION
**Pull request by rmarianski**
@baldur, could you review please?

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 default[:apiaxle][:environment]                            = 'development'
-default[:apiaxle][:version]                                = '1.12.34'
+default[:apiaxle][:version]                                = '1.12.35'
```
Comment (baldur) [inline attributes/default.rb:2]: might be good to use a version number that apiaxle will never use to avoid collision?

File: recipes/api.rb [hunk 2]
```diff
@@ -6,10 +6,11 @@
...
 include_recipe 'apiaxle::_config'
 
-%w(repl api).each do |p|
-  nodejs_npm "apiaxle-#{p}" do
-    version node[:apiaxle][:version]
-  end
+nodejs_npm 'apiaxle-repl' do
+  version node[:apiaxle][:version]
+end
+nodejs_npm 'apiaxle-api' do
```
Comment (baldur) [inline recipes/api.rb]: should we do a conditional here so this recipe still works even if you don't want the forked version

File: attributes/default.rb (hunk 1 above)
Comment (rmarianski) [inline attributes/default.rb:2]: That's just a separate bump to the latest release of apiaxle. Not much changed, and I figured that it would be better to track the latest release version. I probably should have put it in a separate pr, but this was easier and somewhat related anyway.

File: recipes/api.rb (hunk 2 above)
Comment (rmarianski) [inline recipes/api.rb]: Yea, great idea. I updated the pr to support installations with custom urls.
